src/main/resources/application.properties

```diff
@@ -9,6 +9,7 @@ spring.mail.properties.mail.smtp.port=587
 spring.mail.properties.mail.smtp.starttls.enable=true
 spring.mail.properties.mail.smtp.requred.enable=true
 spring.mail.properties.mail.imap.starttls.enable=true
+javastart.agreements-dir=src/main/resources/agreement
```

Comment:
Rozwiązanie w porządku, ale wybór docelowej ścieżki nie jest najlepszy. Jeśli w ramach testów wygenerujesz umowę to ścieżka sugeruje, żeby dodać to do gita. Lepiej wrzucić do jakiegoś folderu, który nie jest w `src`